### ISSUE TITLE
build failiure, missing pcrs header

### ISSUE THREAD

**Issue by apoliakov** [Paradigm4/superfunpack#2]
Working on a particular machine, dev_tools built fine but superfunpack fails to build with this:

```
In file included from pcrs.c:146:
pcrs.h:69:18: error: pcre.h: No such file or directory
In file included from pcrs.c:146:
pcrs.h:134: error: expected specifier-qualifier-list before 'pcre'
pcrs.h:140: warning: struct has no members
pcrs.c: In function 'pcrs_strerror':
pcrs.c:177: error: 'PCRE_ERROR_NOMEMORY' undeclared (first use in this function)
pcrs.c:177: error: (Each undeclared identifier is reported only once
pcrs.c:177: error: for each function it appears in.)
pcrs.c:180: error: 'PCRE_ERROR_NULL' undeclared (first use in this function)
pcrs.c:181: error: 'PCRE_ERROR_BADOPTION' undeclared (first use in this function)
pcrs.c:182: error: 'PCRE_ERROR_BADMAGIC' undeclared (first use in this function)
pcrs.c:183: error: 'PCRE_ERROR_UNKNOWN_NODE' undeclared (first use in this function)
pcrs.c:186: error: 'PCRE_ERROR_NOSUBSTRING' undeclared (first use in this function)
pcrs.c:187: error: 'PCRE_ERROR_NOMATCH' undeclared (first use in this function)
pcrs.c: In function 'pcrs_parse_perl_options':
pcrs.c:239: error: 'PCRE_CASELESS' undeclared (first use in this function)
pcrs.c:240: error: 'PCRE_MULTILINE' undeclared (first use in this function)
pcrs.c:242: error: 'PCRE_DOTALL' undeclared (first use in this function)
pcrs.c:243: error: 'PCRE_EXTENDED' undeclared (first use in this function)
pcrs.c:244: error: 'PCRE_UNGREEDY' undeclared (first use in this function)
pcrs.c: In function 'pcrs_compile_replacement':
pcrs.c:309: warning: implicit declaration of function 'free'
pcrs.c:309: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c: In function 'pcrs_free_job':
pcrs.c:479: error: 'pcrs_job' has no member named 'next'
pcrs.c:480: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:480: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c:480: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:481: error: 'pcrs_job' has no member named 'hints'
pcrs.c:481: error: 'pcrs_job' has no member named 'hints'
pcrs.c:482: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:484: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:484: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:485: error: 'pcrs_job' has no member named 'substitute'
pcrs.c: In function 'pcrs_compile_command':
pcrs.c:595: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c:600: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c: In function 'pcrs_compile':
pcrs.c:655: error: 'pcrs_job' has no member named 'options'
pcrs.c:656: error: 'pcrs_job' has no member named 'flags'
pcrs.c:662: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:662: warning: implicit declaration of function 'pcre_compile'
pcrs.c:662: error: 'pcrs_job' has no member named 'options'
pcrs.c:663: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:674: error: 'pcrs_job' has no member named 'hints'
pcrs.c:674: warning: implicit declaration of function 'pcre_study'
pcrs.c:674: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:687: warning: implicit declaration of function 'pcre_fullinfo'
pcrs.c:687: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:687: error: 'pcrs_job' has no member named 'hints'
pcrs.c:687: error: 'PCRE_INFO_CAPTURECOUNT' undeclared (first use in this function)
pcrs.c:697: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:697: error: 'pcrs_job' has no member named 'flags'
pcrs.c: In function 'pcrs_execute_list':
pcrs.c:747: error: 'pcrs_job' has no member named 'next'
pcrs.c:751: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c: In function 'pcrs_execute':
pcrs.c:814: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:814: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:834: warning: implicit declaration of function 'pcre_exec'
pcrs.c:834: error: 'pcrs_job' has no member named 'pattern'
pcrs.c:834: error: 'pcrs_job' has no member named 'hints'
pcrs.c:836: error: 'pcrs_job' has no member named 'flags'
pcrs.c:847: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:850: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:855: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:860: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:866: warning: implicit declaration of function 'realloc'
pcrs.c:866: warning: incompatible implicit declaration of built-in function 'realloc'
pcrs.c:868: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c:876: error: 'pcrs_job' has no member named 'flags'
pcrs.c:889: error: 'PCRE_ERROR_NOMATCH' undeclared (first use in this function)
pcrs.c:891: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c:903: warning: incompatible implicit declaration of built-in function 'free'
pcrs.c:925: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:928: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:928: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:928: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:929: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:932: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:934: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:936: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:938: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:943: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:944: error: 'pcrs_job' has no member named 'substitute'
pcrs.c:946: error: 'pcrs_job' has no member named 'substitute'
```

**Comment by apoliakov** [comment]
Looks like we're missing a libpcre installation, should include that in the readme perhaps?
Kind of a bummer different plugins need different sets of packages that need root access to install.

**Comment by rvernica** [comment]
Fixed in 6e76f5e852c36f93e0490c0ca9a777a2ceb5134e

**Comment by ksens** [comment]
(Linking here only for future reference -- no action needed)

This issue with `libpcre` `.so` files on Centos 7:
https://github.com/Paradigm4/extra-scidb-libs/issues/21
